docs/source/release_notes.rst

```diff
@@ -9,10 +9,12 @@ Release Notes
     * Fixes
     * Changes
     * Documentation Changes
+        * Remove testing on conda forge in release.md (:pr:`1811`)
```

Comment:
Future release section needs to be uncommented

Reply:
Done